### ISSUE TITLE
PSU 3.6 Can't start dashboard after upgrade

### ISSUE THREAD

**Issue by rwidmark** [ironmansoftware/powershell-universal#1814]
### Steps to Reproduce

When I did revert back to 3.5.5 then it started to work so I'm guessing something is wrong with 3.6?

Did upgrade to PSU 3.6 on two of my machines and now the dashboards won't start.
````
2022-12-13 13:34:01.745 +01:00 [WRN] Failed to groom job
Microsoft.EntityFrameworkCore.DbUpdateConcurrencyException: The database operation was expected to affect 1 row(s), but actually affected 2 row(s); data may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=527962 for information on understanding and handling optimistic concurrency exceptions.
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ThrowAggregateUpdateConcurrencyException(Int32 commandIndex, Int32 expectedRowsAffected, Int32 rowsAffected)
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeResultSetWithoutPropagation(Int32 commandIndex, RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.Consume(RelationalDataReader reader)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(StateManager stateManager, Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.<>c__DisplayClass31_0`2.<Execute>b__0(DbContext context, TState state)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementation[TState,TResult](Func`3 operation, Func`3 verifySucceeded, TState state)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at PowerShellUniversal.SQL.SqlTable`1.Delete(T item) in C:\actions-runner\_work\universal\universal\src\PowerShellUniversal.SQL\Database.cs:line 544
   at UniversalAutomation.GroomService.DeleteJob(Job job) in C:\actions-runner\_work\universal\universal\src\Universal.Server\Services\GroomService.cs:line 217
   at UniversalAutomation.GroomService.ScriptLevelGroom() in C:\actions-runner\_work\universal\universal\src\Universal.Server\Services\GroomService.cs:line 153
````

### Expected behavior

```console
That the dashboards starts
```

### Actual behavior

```console
Don't start
```

### Environment data

PSU 3.6
PSU 7.3
SQL 2019

**Comment by rwidmark** [comment]
So when I did roll back I have started to get the following errors when I'm trying to run scripts:
````
Error executing job: Invalid column name 'DeleteSecret'.
Invalid column name 'Description'.
Invalid column name 'DisableRunAsSupport'.
Invalid column name 'MissingSecret'.
Invalid column name 'Password'.
Invalid column name 'UserName'.
Invalid column name 'Vault'.
````

@adamdriscoll  I think you need to look at this why 3.6.0 don't work when upgrading

**Comment by adamdriscoll** [comment]
@rstolpe We've released version 3.6.1 to address this issue.
